src/Native/Unix/System.Security.Cryptography.Native.Apple/pal_trust.c

```diff
@@ -167,55 +160,102 @@ int32_t AppleCryptoNative_StoreEnumerateUserRoot(CFArrayRef* pCertsOut, int32_t*
 {
     if (pCertsOut != NULL)
         *pCertsOut = NULL;
+    if (pOSStatusOut != NULL)
+        *pOSStatusOut = noErr;
+    if (pCertsOut == NULL || pOSStatusOut == NULL)
```

Comment:
Personally, I like a blank line between the "ensure `out` values are assigned" preamble and the "the function has started now" input validation.

I care enough to say something, but not enough to object :smile:.

Reply:
Added them.